src/js/resize-manager.js

```diff
@@ -114,15 +125,10 @@ class ResizeManager extends Component {
       this.off('load', this.loadListener_);
     }
 
-    if (this.debouncedHandler_) {
-      this.debouncedHandler_.cancel();
-    }
-
     this.ResizeObserver = null;
     this.resizeObserver = null;
     this.debouncedHandler_ = null;
     this.loadListener_ = null;
-    super.dispose();
```

Comment:
We don't really need to `super.dispose` since we turned off touch event reporting.